src/main/java/uk/gov/companieshouse/company_appointments/serialization/LocalDateTimeDeSerializer.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.companieshouse.company_appointments.serialization;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import uk.gov.companieshouse.company_appointments.exception.DeserializationException;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class LocalDateTimeDeSerializer extends JsonDeserializer<LocalDateTime> {
+    public static final String APPLICATION_NAME_SPACE = "company-appointments.api.ch.gov.uk";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(APPLICATION_NAME_SPACE);
+
+    @Override
+    public LocalDateTime deserialize(JsonParser jsonParser, DeserializationContext
+            deserializationContext) {
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter
```

Comment:
`DateTimeFormatter` is thread-safe so it can be declared as a static variable and initialised inline.

Reply:
same as above - duplicate comment.